### DISCUSSION TITLE
Unable to transfer in domain

### DISCUSSION THREAD

**Discussion by quintadonairia** [vercel/community#5213] [Help]
### Summary

Hey 👋 

I want to transfer in my domain (from Namecheap) to Vercel.

I've followed the [guide](https://vercel.com/guides/how-do-i-transfer-my-domain-to-vercel). Can confirm the domain is eligible, in unlocked, and I got the Auth Code from Namecheap.

However, when I click on "Transfer In", the following error shows up:

I do have this domain ("quintadonairia.com") on my account, which is a third-party domain from Namecheap. 
I tried to remove the domain briefly, and transfer in: same error shows up.

Am I missing something basic here? My intuition would be that, having the third-party domain in my account, I would be able to transfer it in to Vercel Domains.

Thank you 🙏 

### Steps to Reproduce

<img width="450" alt="image" src="https://github.com/vercel/community/assets/106924627/38644423-4b31-4248-ba6c-435afb0ff151">

**Comment by tomytubert** [comment]
I have the same problem. Did you solve it?

**Reply by ReloadGames** [reply]
Me too, any support here?
